source/fe/fe_nedelec_sz.cc

```diff
@@ -162,13 +162,25 @@ FE_NedelecSZ<dim, spacedim>::shape_value(const unsigned int /*i*/,
 template <int dim, int spacedim>
 double
 FE_NedelecSZ<dim, spacedim>::shape_value_component(
-  const unsigned int /*i*/,
-  const Point<dim> & /*p*/,
-  const unsigned int /*component*/) const
+  const unsigned int i,
+  const Point<dim>  &p,
+  const unsigned int component) const
 {
-  // Not implemented yet:
-  DEAL_II_NOT_IMPLEMENTED();
-  return 0.;
+  AssertIndexRange(i, this->n_dofs_per_cell());
+  AssertIndexRange(component, dim);
+
+  std::unique_ptr<
+    typename dealii::FiniteElement<dim, spacedim>::InternalDataBase>
+    data_ptr = std::make_unique<InternalData>();
```

Comment:
Oh, this is expensive. Would it be useful to mention that these functions are expensive in the documentation of this class?

Reply:
I've added a remark. And reworked the Doxygen description of those functions.